.github/workflows/checkAndSubmitAddonMetadata.yml

```diff
@@ -179,6 +179,137 @@ jobs:
+      with:
+        repository-id: $GITHUB_REPOSITORY_ID
+        category-id: 'DIC_kwDODeB9Us4CaB4B'
+        title: Reviews for ${{ steps.getAddonNameAndVersion.outputs.addonName }} add-on
+        body: |
+          Community add-on reviews for {{ steps.getAddonNameAndVersion.outputs.addonName }}.
+          Please use threaded replies and avoid email responses.
+          Reviews for specific versions of the add-on can be added in response to automated comments for each version.
+          Reviews should abide by the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
+    - name: Create review comment
+      if: ${{ steps.checkDiscussion.outcome }} == 'success'
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const fs = require('fs')
+          const contents = fs.readFileSync('discussions.json')
+          const data = JSON.parse(contents)
+          const addonName = "${{ steps.getAddonNameAndVersion.outputs.addonId }}"
+          const addonVersion = "${{ steps.getAddonNameAndVersion.outputs.addonVersion }}"
+          const discussionId = data[addonId]["discussionId"]
+          const mutation = `mutation {
+            addDiscussionComment(input: {body: "Reviews for version ${addonVersion}", discussionId: "${discussionId}", clientMutationId: "GitHub Script action"}) {
+              clientMutationId
+              comment {
+                body
+            }
+            }
+          }`;
+          github.graphql(mutation)
+    - name: Add add-on ID and discussion data to json file
+      if: ${{ steps.checkDiscussion.outcome }} == 'failure'
+      run: |
+        addonId=$(
+          echo ${{ needs.getAddonFilename.outputs.addonFileName }} \
+          | sed -r "s|addons/(.*)/.*\.json|\1|g"
+        )
+        discussionId=$(
+          echo ${{ steps.createDiscussion.outputs.discussion-id }}
+        )
+        discussionUrl=$(
+          echo ${{ steps.createDiscussion.outputs.discussion-url }}
+        )
+        jqCode="
+        .[\"$addonId\"].discussionId = \"$discussionId\"
+        | .[\"$addonId\"].discussionUrl = \"$discussionUrl\"
+        "
+  
+        mv discussions.json discussions.old.json
+        jq -e "$jqCode" discussions.old.json > discussions.json
+        jqExitCode=$?
+        rm discussions.old.json
+        exit $jqExitCode
+    - name: Add discussion URL to metadata
+      run: |
+        addonFilename=$(
+          echo ${{ needs.getAddonFilename.outputs.addonFileName }}
+        )
+        discussionUrl=$(
+          echo ${{ steps.createDiscussion.outputs.discussion-url }}
+        )
+        jqCode="
+        .[\"discussionUrl\"] = \"$discussionUrl\"
+        "
+  
+        mv $addonFilename $addonFilename.old.json
+        jq -e "$jqCode" $addonFilename.old.json > $addonFilename
+        jqExitCode=$?
+        rm $addonFilename.old.json
+        exit $jqExitCode
+    - name: Commit and push
```

Comment:
it seems there are two commit and push actions for the same pull request. This should probably be one commit that always happens with the message "update discussion URL"